src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -183,14 +183,24 @@ export const DetailsForm: React.FC<DetailsFormProps> = ({
             user
           );
 
-          if (sovereignVenueId)
+          //@debt Create separate function that updates the userStatuses separately by venue id.
+          if (sovereignVenueId && sovereignVenue)
             await updateVenue(
               {
-                ...(vals as VenueInput),
+                name: sovereignVenue.name,
+                subtitle:
+                  sovereignVenue.config?.landingPageConfig.subtitle ?? "",
+                description:
+                  sovereignVenue.config?.landingPageConfig.description ?? "",
+                adultContent: sovereignVenue.adultContent ?? false,
+                profile_questions: sovereignVenue.profile_questions,
+                code_of_conduct_questions:
+                  sovereignVenue.code_of_conduct_questions,
```

Comment:
Why does it update sovereignVenue, not the venue?

Reply:
User statuses are stored in the sovereignVenue that way the whole world can share them. This is why after I update the current venue I need to update the sovereignVenue with the userStatuses only.

Reply:
Just to be clear. What if you don't pass the field here? Will it be erased?

Reply:
checked the logic, shouldn't do it. Only for the specific fields such as this
![image](https://user-images.githubusercontent.com/63194656/125281648-28077200-e31f-11eb-846b-801942df47ad.png)

Reply:
checked the logic, shouldn't do it. Only for the specific fields such as this
![image](https://user-images.githubusercontent.com/63194656/125281648-28077200-e31f-11eb-846b-801942df47ad.png)

Reply:
@mike-lvov It's the types that actually force me to pass these fields, I can make them optional. If that's okey.